src/themes.php

```diff
@@ -1046,4 +1046,16 @@
         "sideLabels" => "#000000",
         "dates" => "#000000",
     ],
+    "violet-dark" => [
+      "background" => "#000000",
+      "border" => "#FF0089",
+      "stroke" => "#FF0089",
+      "ring" => "#FF0089",
+      "fire" => "#FF0089",
+      "currStreakNum" => "#FF0089",
+      "sideNums" => "#FF0089",
+      "currStreakLabel" => "#FF0089",
+      "sideLabels" => "#FF0089",
+      "dates" => "#FF0089",
```

Comment:
The spacing is a bit different
```suggestion
        "background" => "#000000",
        "border" => "#FF0089",
        "stroke" => "#FF0089",
        "ring" => "#FF0089",
        "fire" => "#FF0089",
        "currStreakNum" => "#FF0089",
        "sideNums" => "#FF0089",
        "currStreakLabel" => "#FF0089",
        "sideLabels" => "#FF0089",
        "dates" => "#FF0089",
```